paddle/fluid/imperative/layer.h

```diff
@@ -380,6 +390,10 @@ class OpBase : public std::enable_shared_from_this<OpBase> {
     return grad_pending_ops_;
   }
 
+  void SetGradPendingOps(std::vector<OpBase*>* vec_temp) {
```

Comment:
You can use `std::vector<OpBase *> vec_tmp` as parameter instead of a pointer.

Reply:
Using a porint to avoid copy